pegasus/helpers/section_api_helpers.rb

```diff
@@ -9,8 +9,9 @@ class DashboardStudent
   def self.fetch_user_students(user_id)
     Dashboard.db[:users].
       join(:followers, :student_user_id=>:users__id).
+      join(Sequel.as(:users, :users_students), :id=>:followers__student_user_id).
```

Comment:
Yes. Apparently, though stuff works magically in dashboard land (using ActiveRecord), it doesn't work magically in pegasus land. So these changes are essentially bug fixes in our API implementation.